From 981ac7d03d05da6cbdc354970bb37342f077c9ee Mon Sep 17 00:00:00 2001
From: Mike Sarahan <msarahan@gmail.com>
Date: Wed, 15 Jan 2020 09:11:35 -0600
Subject: [PATCH 3/4] remove PATH pth hack

---
 pywin32.pth | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/pywin32.pth b/pywin32.pth
index b57c496..0b75a1e 100644
--- a/pywin32.pth
+++ b/pywin32.pth
@@ -2,6 +2,7 @@
 win32
 win32\lib
 Pythonwin
-# And some hackery to deal with environments where the post_install script
-# isn't run.
-import pywin32_bootstrap
+# This breaks any software involving a feedback loop involving PATH and Python
+# One example of such software is the Anaconda Distribition's conda package
+# manager, so we do not do it. We fixed this in a slightly better way anyway.
+# import pywin32_bootstrap
-- 
2.20.1.windows.1

