From 8b76c76826d02724fae8d843a9b71ca4fd620725 Mon Sep 17 00:00:00 2001
From: Mike Sarahan <msarahan@gmail.com>
Date: Wed, 15 Jan 2020 09:14:11 -0600
Subject: [PATCH 4/4] remove JOB_OBJECT_RESERVED_LIMIT_VALID_FLAGS

---
 win32/src/win32job.i | 1 -
 1 file changed, 1 deletion(-)

diff --git a/win32/src/win32job.i b/win32/src/win32job.i
index d773a89..5a27892 100644
--- a/win32/src/win32job.i
+++ b/win32/src/win32job.i
@@ -459,7 +459,6 @@ PyObject *PySetInformationJobObject(PyObject *self, PyObject *args)
 
 #define JOB_OBJECT_BASIC_LIMIT_VALID_FLAGS JOB_OBJECT_BASIC_LIMIT_VALID_FLAGS
 #define JOB_OBJECT_EXTENDED_LIMIT_VALID_FLAGS JOB_OBJECT_EXTENDED_LIMIT_VALID_FLAGS
-#define JOB_OBJECT_RESERVED_LIMIT_VALID_FLAGS JOB_OBJECT_RESERVED_LIMIT_VALID_FLAGS
 
 #define JOB_OBJECT_UILIMIT_NONE JOB_OBJECT_UILIMIT_NONE
 #define JOB_OBJECT_UILIMIT_HANDLES JOB_OBJECT_UILIMIT_HANDLES
-- 
2.20.1.windows.1

