diff --git a/setup.py b/setup.py
index 8c397e1e..88d55efe 100644
--- a/setup.py
+++ b/setup.py
@@ -709,11 +709,10 @@ def _fixup_sdk_dirs(self):
 
     def _why_cant_build_extension(self, ext):
         # Return None, or a reason it can't be built.
-        # Exclude exchange 32-bit utility libraries from 64-bit
-        # builds. Note that the exchange module now builds, but only
-        # includes interfaces for 64-bit builds.
-        if self.plat_name == 'win-amd64' and ext.name == 'exchdapi':
-            return "No 64-bit library for utility functions available."
+        if ext.name in {'exchdapi', 'exchange'}:
+            # Dont need these modules, and they require headers not present in
+            # the windows SDK
+            return ext.name + " module is disabled"
         if get_build_version() >=14:
             if ext.name == 'exchange':
                 ext.libraries.append('legacy_stdio_definitions')
@@ -891,7 +890,7 @@ def lookupMfcInVisualStudio(self, mfc_version, mfc_libraries):
             # Should have the same length - if not we lost a file!
             if len(mfc_files) is not len(mfc_contents):
                 mfc_contents = []
-        
+
         return mfc_contents
 
     def lookupMfcInWinSxS(self, mfc_version, mfc_libraries):
@@ -923,7 +922,7 @@ def lookupMfcInWinSxS(self, mfc_version, mfc_libraries):
                 print("Could not find WinSxS directory in %WINDIR%.")
         else:
             print("Windows directory not found!")
-        
+
         return mfc_contents
 
     def build_extensions(self):
