diff --git a/isapi/src/pyISAPI_messages.mc b/isapi/src/PyISAPI_messages.mc
similarity index 100%
rename from isapi/src/pyISAPI_messages.mc
rename to isapi/src/PyISAPI_messages.mc
diff --git a/isapi/src/PythonEng.cpp b/isapi/src/PythonEng.cpp
index 008f1d4..264a0e1 100644
--- a/isapi/src/PythonEng.cpp
+++ b/isapi/src/PythonEng.cpp
@@ -31,7 +31,7 @@
 #include "PythonEng.h"
 #include "pyExtensionObjects.h"
 #include "pyFilterObjects.h"
-#include "pyISAPI_messages.h"
+#include "PyISAPI_messages.h"
 
 extern HINSTANCE g_hInstance;
 extern bool g_IsFrozen;
diff --git a/setup.py b/setup.py
index dffe096..b4dbeb3 100644
--- a/setup.py
+++ b/setup.py
@@ -1686,7 +1686,7 @@ win32_extensions += [
 ]
 win32_extensions += [
     WinExt_win32('servicemanager',
-           sources = ["win32/src/PythonServiceMessages.mc", "win32/src/PythonService.cpp"],
+           sources = ["win32/src/APythonServiceMessages.mc", "win32/src/PythonService.cpp"],
            extra_compile_args = ['-DPYSERVICE_BUILD_DLL'],
            libraries = "user32 ole32 advapi32 shell32",
            windows_h_version = 0x500,
@@ -2250,10 +2250,10 @@ other_extensions.append(
     WinExt_ISAPI('PyISAPI_loader',
        sources=[os.path.join("isapi", "src", s) for s in
                """PyExtensionObjects.cpp PyFilterObjects.cpp
-                  pyISAPI.cpp pyISAPI_messages.mc
+                  pyISAPI.cpp PyISAPI_messages.mc
                   PythonEng.cpp StdAfx.cpp Utils.cpp
                """.split()],
-       # We keep pyISAPI_messages.h out of the depends list, as it is
+       # We keep PyISAPI_messages.h out of the depends list, as it is
        # generated and we aren't smart enough to say *only* the .cpp etc
        # depend on it - so the generated .h says the .mc needs to be
        # rebuilt, which re-creates the .h...
diff --git a/win32/src/PythonServiceMessages.mc b/win32/src/APythonServiceMessages.mc
similarity index 100%
rename from win32/src/PythonServiceMessages.mc
rename to win32/src/APythonServiceMessages.mc
diff --git a/win32/src/PythonService.cpp b/win32/src/PythonService.cpp
index 54e29dd..3c3d849 100644
--- a/win32/src/PythonService.cpp
+++ b/win32/src/PythonService.cpp
@@ -46,7 +46,7 @@ static void ReportPythonError(DWORD);
 static BOOL ReportError(DWORD, LPCTSTR *inserts = NULL, WORD errorType = EVENTLOG_ERROR_TYPE);
 static void CheckRegisterEventSourceFile();
 
-#include "PythonServiceMessages.h"
+#include "APythonServiceMessages.h"
 
 // Useful for debugging problems that only show themselves when run under the SCM
 #define LOG_TRACE_MESSAGE(msg)                                                   \
