diff --git a/tests/test_api.py b/tests/test_api.py
index b32662e..d44e7d5 100644
--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -183,7 +183,7 @@ def test_create_package_with_uncommon_conditions_captures_all_content(testing_wo
             assert stat.st_nlink == 1
 
 
-@pytest.mark.skipif(datetime.now() <= datetime(2020, 12, 1), reason="Don't understand why this doesn't behave.  Punt.")
+@pytest.mark.skipif(datetime.now() <= datetime(2021, 12, 1), reason="Don't understand why this doesn't behave.  Punt.")
 def test_secure_refusal_to_extract_abs_paths(testing_workdir):
     with tarfile.open('pinkie.tar.bz2', 'w:bz2') as tf:
         open('thebrain', 'w').close()
